rct229/ruleset_functions/baseline_systems/is_baseline_system_2.py

```diff
@@ -76,7 +76,7 @@ def is_baseline_system_2(rmi_b, hvac_b_id, terminal_unit_id_list, zone_id_list):
         and are_all_terminal_heat_sources_none_or_null(rmi_b, terminal_unit_id_list)
         and are_all_terminal_cool_sources_none_or_null(rmi_b, terminal_unit_id_list)
         and are_all_terminal_fans_null(rmi_b, terminal_unit_id_list)
-        and are_all_terminal_supplies_ducted(rmi_b, terminal_unit_id_list)
+        and not are_all_terminal_supplies_ducted(rmi_b, terminal_unit_id_list)
```

Comment:
Oh wow.  I don't think I've been careful enough in my reviews to notice any of these functions calls needing to be False.  Have you  rechecked (or could you recheck) the other baseline_systems functions for this possibility?

Reply:
Of course, Charlie. I examined all other baseline_systems functions carefully, and I didn't find any function that we flipped the True/False.